src/jq.test.js

```diff
@@ -74,6 +75,31 @@ describe('jq core', () => {
       })
   })
 
+  it('should catch errors from child process stdin', done => {
+    // This is a very specific case of error, only triggered if:
+    // 1) The jq process exits early (e.g. due to an invalid filter)
+    // AND
+    // 2) We are trying to send a large amount of data over stdin,
+    //    (which will take longer to do than jq takes to exit).
+    const largeJsonString = JSON.parse(readFileSync(PATH_LARGE_JSON_FIXTURE))
+    run(FILTER_INVALID, largeJsonString, { input: 'json' })
+      .then(result => {
+        done('Expected an error to be thrown from child process stdin')
```

Comment:
This shouldn't be here otherwise the test always pass green. Can you remove the done call, and maybe throw or handle this in another fashion?

Reply:
Calling done without any arguments passes a test, but calling it _with_ arguments fails the test. Per the [Mocha documentation](https://mochajs.org/#asynchronous-code), "This callback accepts both an Error instance (or subclass thereof) or a falsy value; anything else is invalid usage and throws an error (usually causing a failed test)."

I've tried this out locally - making the `run(...)` Promise resolve so that this `done` call is hit, and the test does fail in that case. It would be more proper to wrap this error message in an `Error`, but either way this done call does fail the test.